eZ/Bundle/EzPublishCoreBundle/Command/UpdateTimestampsToUTCCommand.php

```diff
@@ -0,0 +1,529 @@
+<?php
+/**
+ * This file is part of the eZ Publish Kernel package.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved
+ * @license For full copyright and license information view LICENSE file distributed with this source code
+ */
+namespace eZ\Bundle\EzPublishCoreBundle\Command;
+
+use DateTime;
+use DateTimeZone;
+use Doctrine\DBAL\Connection;
+use RuntimeException;
+use Symfony\Component\Process\Process;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Helper\ProgressBar;
+use Symfony\Component\Console\Question\ConfirmationQuestion;
+use Symfony\Component\Process\PhpExecutableFinder;
+use PDO;
+
+class UpdateTimestampsToUTCCommand extends ContainerAwareCommand
+{
+    const DEFAULT_ITERATION_COUNT = 100;
+    const EZDATE_ONLY_MODE = 'date';
+    const EZDATETIME_ONLY_MODE = 'datetime';
+    const ALL_MODE = 'all';
```

Comment:
I think you can do the following:
```
const MODES = [
    'date' => ['ezdate'],
    'datetime' => ['ezdatetime'],
    'all' => ['ezdate', 'ezdatetime'],
];
```

Simplifies validation of option, and simplifies mapping in `getField[TypeIdentifier]s()`.
Can even use this as source for help text on the option to solve all open comments ;)